### ISSUE TITLE
SWRL rules dont appear.

### ISSUE THREAD

**Issue by vChavezB** [dgarijo/Widoco#651]
I am trying to generate the SWRL rules I declare in my ontology but they dont apper after I generate the project.

The rules section appears empty with just a the title "4.6 Rules" and a blue rectangle

![grafik](https://github.com/dgarijo/Widoco/assets/47216966/378a355f-efaa-44a2-a9f3-c56a0e5a4a87)

Do you have an example ontology that is  known to be working and I can use to check if its just on my side?

Here is an excerpt of how I declare my SWRL rules (generated with protege)

```turtle
@prefix swrl: <http://www.w3.org/2003/11/swrl#> .
[ <http://swrl.stanford.edu/ontologies/3.3/swrla.owl#isRuleEnabled> "true"^^xsd:boolean ;
   rdfs:label "My rule" ;
   rdf:type swrl:Imp ;
   swrl:body [ rdf:type swrl:AtomList ;
               rdf:first [ rdf:type swrl:ClassAtom ;
                           swrl:classPredicate :ApplicationDataObject ;
                           swrl:argument1 :ao
                         ] ;
               rdf:rest [ rdf:type swrl:AtomList ;
                          rdf:first [ rdf:type swrl:DatavaluedPropertyAtom ;
                                      swrl:propertyPredicate :isProcessData ;
                                      swrl:argument1 :ao ;
                                      swrl:argument2 "true"^^xsd:boolean
                                    ] ;
                          rdf:rest rdf:nil
                        ]
             ] ;
   swrl:head [ rdf:type swrl:AtomList ;
               rdf:first [ rdf:type swrl:ClassAtom ;
                           swrl:classPredicate :ProcessDataObject ;
                           swrl:argument1 :ao
                         ] ;
               rdf:rest rdf:nil
             ]
 ] .
```

**Comment by vChavezB** [comment]
Ok I made a tmp fix, perhaps you can check if it looks fine. The rules are now displayed. However as you see the first element is "null". 

![grafik](https://github.com/dgarijo/Widoco/assets/47216966/90644b52-4cb7-4f94-a8d7-47498b18c20c)

**Comment by dgarijo** [comment]
Thanks a lot. So far, I think no swrl rules were considered, as I did not have a use case for them (I may have simplified them from the XSLT output). I will need to review the PR, since in the `develop` branch I was considering other types of rules I had been asked to include (but not SWRL). This is a great addition!

**Comment by vChavezB** [comment]
I read the code again and the xlst sources. So if I understand correctly the "rules" extraction uses annotations from owl Individuals or which type of rules does it parse.

https://github.com/dgarijo/Widoco/blob/4da92b23995c37a95662bce03a8a174ccaa44f7f/src/main/resources/lode/extraction.xsl#L2334

 I did not found a sample  ontology that uses this transformation. This would be helpful to know as it was not clear to me. I will update the PR and refine it.

**Comment by dgarijo** [comment]
@vChavezB sorry, the rules stuff I added is for very specific owl annotations for a specific project I am doing.

The SWRL module you added recovers the original SWRL annotations, and I think they should be added, but separately. 
Please just ignore the module I did for annotations of rules and use a new one with the id you gave, as you were doing. Thanks in advance!

**Comment by dgarijo** [comment]
This issue should now be closed and will be up in the next release See #654 for more info.
Thanks @vChavezB  for your work!!
